fpm/src/fpm/cmd/new.f90

```diff
@@ -137,34 +267,91 @@ subroutine cmd_new(settings)
         call warnwrite(join_path(settings%name, 'app/main.f90'), littlefile)
     endif
 
+    if(settings%with_full)then
+       tomlfile=[character(len=80) ::  tomlfile,   &
+       &'[dependencies]                                                                  ', &
```

Comment:
Could be good to add a commented-out remote dependency entry as an example. Would be nice to a fortran-lang repo to use here.